stylesheets/back-office.css

```diff
@@ -721,4 +721,9 @@ table {
 
   .isSplitPackingClass {
     page-break-after: always;
+}
+ 
+.mz-printable-order-packingslip .slip-product-image {
```

Comment:
Include pick wave or pick sheet in the class name,
As these classes are used by customers to customize themes as per their needs,
So including feature related names in class helps them to recognize them quickly,
--> Class name suggestion
.mz-printable-order-pickwave-packingslip .pickwave-slip-product-image

Reply:
Resolved